alf/layers.py

```diff
@@ -286,8 +286,11 @@ def reset_parameters(self):
         if self._use_bias:
             nn.init.constant_(self._linear.bias.data, self._bias_init_value)
 
-    def forward(self, inputs):
-        return self._activation(self._linear(inputs))
+    def forward(self, inputs, info=None):
```

Comment:
This seems quite hacky. The other layers don't have this signature.

Reply:
Yes, agree.  Let me remove this for now.